data/lang/ui-core/en.json

```diff
@@ -1735,6 +1735,14 @@
     "description": "",
     "message": "Mute"
   },
+  "MUTE_ON_FOCUS": {
+    "description": "Setting for muting audio when the game window loses focus.",
+    "message": "Mute on Focus"
```

Comment:
```suggestion
    "message": "Mute on Window Focus Lost"
```

Reply:
Fixed in  3bd058eca69e9ab8b578c0fcff481dd21a67f50e (to be squashed)